Packs/HatchingTriage/pack_metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "Hatching Triage",
+    "description": "Scalable malware sandbox",
+    "support": "xsoar",
```

Comment:
Is there a process to submit this as official content supported by XSOAR?

Reply:
Hi @mhouston007, You can open an issue for the content team that you want the team to take ownership on this integration however it will need to get into the roadmap. If you prefer that until then the pack **will not** appear as community-supported it is legit, we could close this and wait until then. 
Let us know.

Reply:
Updated to community support